### DISCUSSION TITLE
How to Show the user avatar according to its status.

### DISCUSSION THREAD

**Discussion by devhoussam1998** [filamentphp/filament#9585] [Help]
Suppose I have a filament panel named **app** featuring various user statuses such as **Trial** and **Premium**. I aim to display a distinct avatar corresponding to each user status. How can I accomplish this using Filament?

https://filamentphp.com/docs/3.x/panels/users#using-a-different-avatar-provider

**Comment by coolsam726** [comment]
How about overriding the `getFilamentAvatarUrl` method?
https://filamentphp.com/docs/3.x/panels/users#setting-up-user-avatars
